### ISSUE TITLE
Failure when debugging Spring Boot pod in VSCode

### ISSUE THREAD

**Issue by Setre14** [metalbear-co/mirrord-vscode#32]
### Bug Description

Mirrord can't connect to a Spring boot application from the VSCode extension. Similar to https://github.com/metalbear-co/mirrord-intellij/issues/42 for the Intellij plugin

### Steps to Reproduce

1. Try to debug a Spring boot application using mirrod + VSCode extension

### Relevant Logs

```shell
ERROR: transport error 202: connect failed: Connection refused
ERROR: JDWP Transport dt_socket failed to initialize, TRANSPORT_INIT(510)
JDWP exit error AGENT_ERROR_TRANSPORT_INIT(197): No transports initialized [debugInit.c:750]
```

### Your operating system and version

Fedora 38

### Local process

-

**Comment by aviramha** [comment]
Thanks for reporting this. We'll get it sorted ASAP.

**Comment by eyalb181** [comment]
Hey @Setre14, we just released version 3.49.1 of the VS Code extension, which should resolve this. Please let us know if it recurs!
